src/data/full-quiz.ts

```diff
@@ -7240,5 +7240,25 @@ const fullQuiz = [
       "According to the Scrum Guide, the Product Backlog is never complete. Initially, it contains only the best-understood requirements.",
     Link: "https://www.freecodecamp.org/news/scrum-the-hard-parts-2-sprint-harder/",
   },
+  {
+    Question:
+      "Choose the correct option?",
+    Answer: "Little O (o()) describes the upper bound excluding the exact bound",
+    Distractor1: "Big O (O()) describes the lower bound of the complexity.",
+    Distractor2: "Omega (Ω()) describes the upper bound of the complexity",
+    Distractor3: "Theta (Θ()) describes the lower bound excluding the exact bound",
+    Explanation: "Big O (O()) - Upper Bound ,Omega (Ω()) - Lower Bound , Theta (Θ()) - exact bound & Little O (o()) - describes the upper bound excluding the exact bound of the complexity ",
```

Comment:
The explanation can be cleaned up a little bit. Let's just show the explanation for Little O. If they want to read more then they can click on the link for the article. 

```suggestion
    Explanation: "Little O (o()) describes the upper bound excluding the exact bound of the complexity ",
```